test/app/domain/use_case/update_user_profile_use_case_test.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:dartz/dartz.dart';
+import 'package:flash_cards/app/utils/enums/errors.dart';
+import 'package:flash_cards/domain/use_case/update_user_profile_use_case.dart';
+import 'package:flash_cards/domain/utils/failure.dart';
+import 'package:flash_cards/domain/utils/success.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+
+import '../../../mocked_data.dart';
+import '../../../mocks.mocks.dart';
+
+void main() {
+  late MockDatabaseRepository mockDatabaseRepository;
+  late UpdateUserProfileUseCase updateUserProfileUseCase;
+
+  setUp(() {
+    mockDatabaseRepository = MockDatabaseRepository();
+    updateUserProfileUseCase = UpdateUserProfileUseCase(mockDatabaseRepository);
+  });
+
+  test('UserProfileUseCas updates information about user success', () async {
+    when(mockDatabaseRepository.updateUserProfile(mockedUserProfileEntity)).thenAnswer((_) async => const Right(Success()));
```

Comment:
```suggestion
    when(mockDatabaseRepository.updateUserProfile(any)).thenAnswer((_) async => const Right(Success()));
```
Add any in every test:
1. on when method
2. on verify method